modules/bootstrap-aws-load-balancer-operator.adoc

```diff
@@ -11,6 +11,13 @@
 
 .Procedure
 
+. Create the `aws-load-balancer-operator` namespace:
```

Comment:
```suggestion
. Create the `aws-load-balancer-operator` namespace by running the following command:
```

Reply:
I think it's recommended in the OS docs guidelines to have an instruction for user to run the command for translation/localization purposes.

Reply:
Addressed